src/allmydata/test/cli/wormholetesting.py

```diff
@@ -141,8 +141,8 @@ def _verify():
     """
     # Poor man's interface verification.
 
-    a = getargspec(create)
-    b = getargspec(MemoryWormholeServer.create)
+    a = getfullargspec(create)
+    b = getfullargspec(MemoryWormholeServer.create)
```

Comment:
I think this will fail.  There are some more fixes related to this in #1261 which hopefully will land soon.

Reply:
It passed...